File: Converge/Models/EventRecipient.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using Microsoft.Graph;

namespace Converge.Models
{
    public class EventRecipient
    {
        public string Name { get; set; }

        public string EmailAddress { get; set; }

        public AttendeeType Type { get; set; }
    }
}

File: Converge/ClientApp/src/types/EventRecipient.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

import * as MicrosoftGraph from "@microsoft/microsoft-graph-types";

interface EventRecipient {
  name?: MicrosoftGraph.NullableOption<string>;
  emailAddress?: MicrosoftGraph.NullableOption<string>;
  type: MicrosoftGraph.AttendeeType
}

export default EventRecipient;
